packages/firestore/src/local/target_data.ts

```diff
@@ -20,59 +20,58 @@ import { Target } from '../core/target';
 import { ListenSequenceNumber, ProtoByteString, TargetId } from '../core/types';
 import { emptyByteString } from '../platform/platform';
 
-/** An enumeration of the different purposes we have for queries. */
+/** An enumeration of the different purposes we have for targets. */
 export enum QueryPurpose {
```

Comment:
Did you consider renaming to TargetPurpose ?  I don't feel strongly, but I think it might make sense.